develop/migration_mt_schema.rst

```diff
@@ -18,7 +18,7 @@ We are happy to help review your environment to be sure that the ideal distribut
 Identify types of tables
 ------------------------
 
-Once a distribution key is identified, review the schema to identify how each table will be handled and whether any modifications to table layouts will be required. We typically advise tracking this with a spreadsheet, and have created a `template <https://examples.citusdata.com/citus-migration-plan.xlsx>`_ you can use.
+Once a distribution key is identified, review the schema to identify how each table will be handled and whether any modifications to table layouts will be required. We typically advise tracking this with a spreadsheet, and have created a `template <https://docs.google.com/spreadsheets/d/1jYlc22lHdP91pTrb6s35QfrN9nTE1BkVJnCSZeQ7ZmI/edit>`_ you can use.
```

Comment:
@saicitus or @colton-citus : do you have some nice example data for this spreadsheet? Or should I use the same tables that we have in the migration docs?

Reply:
I'm not sure if we want to pre-fill the spreadsheet, but if you're sure that we do, just use the same tables you already use as examples.